packages/website/components/contexts/plansContext.js

```diff
@@ -17,77 +17,94 @@
  * @typedef {'earlyAdopter'} EarlyAdopterPlanId
  */
 
+/**
+ * @typedef {object} StripeTier
+ * @property {number|null} flat_amount
+ * @property {number|null} unit_amount
+ * @property {number|null} up_to
+ */
+
 /**
  * @typedef {object} Plan
- * @property {StoragePrice|EarlyAdopterPlanId} id
- * @property {string} title
- * @property {string} description
- * @property {string} price
- * @property {string} baseStorage
- * @property {string} additionalStorage
- * @property {string} bandwidth
- * @property {string} blockLimit
+ * @property {string} id
+ * @property {import("@stripe/stripe-js").Metadata} metadata
```

Comment:
I think it was better before when this was using a plan type that doesn't depend on stripe, and isn't as vaguely defined as `Stripe.Metadata`

Reply:
updated (see [here](https://github.com/web3-storage/web3.storage/pull/2152#discussion_r1065217803))